### PR TITLE
Cleanup

### LINKED ISSUE #19: PEP8
`happi` should conform entirely with PEP8

### PR DESCRIPTION
**Pull request by teddyrendahl**
## Description

A bunch of fairly small, but necessary changes to make things a little nicer.
* The entire library satisfies PEP8 requirements
* The tests no longer strictly require `mongomock`
* No longer need to import `six` as a requirement
* Documentation Cleanup

## Motivation and Context

Closes #19 

## How Has This Been Tested?

Same tests, still passing

### CODE REVIEW

File: docs/source/device.rst [hunk 1]
```diff
@@ -42,42 +42,17 @@ Device container. This section documents the entries when they either do not
...
-.. autoclass:: happi.containers.PulsePicker
-
-LODCM
-+++++
-.. autoclass:: happi.containers.LODCM
-
-MovableStand
-+++++++++++++
-.. autoclass:: happi.containers.MovableStand
+.. autosummary::
```
Comment (ZLLentz) [inline docs/source/device.rst]: Sphinx keyword of the year right here

File: happi/client.py [hunk 2]
```diff
@@ -10,7 +10,6 @@
 ###############
 # Third Party #
 ###############
```
Comment (ZLLentz) [inline happi/client.py]: This is why I don't like the import comment headers. Look at the big "we don't use 3rd party modules" banner it leaves. https://www.python.org/dev/peps/pep-0008/#imports specifies only a blank line between each import group.

File: docs/source/device.rst (hunk 1 above)
Comment (teddyrendahl) [reply docs/source/device.rst]: Yup!

File: happi/client.py (hunk 2 above)
Comment (teddyrendahl) [reply happi/client.py]: I'll remove these in the next PR

**Pull request merged**